src/lib/InstantSearch.js

```diff
@@ -136,10 +138,11 @@ Usage: instantsearch({
   }
 
   _init(state, helper) {
+    const onHistoryChange = this._onHistoryChange || function() {};
```

Comment:
Why no test the default value in the upper else (like we do for _createURL), would look more consisten I feel.